var/spack/repos/builtin/packages/py-reportlab/package.py

```diff
@@ -20,7 +22,19 @@ class PyReportlab(PythonPackage):
     # - pip, provided by py-pip
     extends("python", ignore=r"bin/.*")
 
+    depends_on("python@3.7:3", when="@3.6.9:", type=("build", "run"))
     depends_on("py-setuptools@2.2:", type="build")
     depends_on("py-pip@1.4.1:", type="build")
+    depends_on("pil@9:", when="@3.6.10:", type=("build", "run"))
     depends_on("pil@2.4.0:", type=("build", "run"))
     depends_on("freetype")
+
+    def patch(self):
+        filter_file(
+            "[FREETYPE_PATHS]",
+            "[FREETYPE_PATHS]\n"
+            + "lib={}\n".format(self.spec["freetype"].prefix.lib)
+            + "inc={}\n".format(self.spec["freetype"].prefix.include),
```

Comment:
```suggestion
            + "lib={}\n".format(self.spec["freetype"].libs.directories[0])
            + "inc={}\n".format(self.spec["freetype"].headers.directories[0]),
```
This will work even if libs are in lib64 instead.

Reply:
Thanks.